doc/classes/ViewportTexture.xml

```diff
@@ -7,6 +7,7 @@
 		A [ViewportTexture] provides the content of a [Viewport] as a dynamic [Texture2D]. This can be used to combine the rendering of [Control], [Node2D] and [Node3D] nodes. For example, you can use this texture to display a 3D scene inside a [TextureRect], or a 2D overlay in a [Sprite3D].
 		To get a [ViewportTexture] in code, use the [method Viewport.get_texture] method on the target viewport.
 		[b]Note:[/b] A [ViewportTexture] is always local to its scene (see [member Resource.resource_local_to_scene]). If the scene root is not ready, it may return incorrect data (see [signal Node.ready]).
+		[b]Note:[/b] The setup of [ViewportTexture] configured in tscn is extremely dependent on its local scene, please make sure [member Resource.resource_local_to_scene] is enabled for all resources on the absolute [NodePath] to the resource.
```

Comment:
```suggestion
		[b]Note:[/b] The setup of [ViewportTexture] configured in a scene is extremely dependent on its local scene, please make sure [member Resource.resource_local_to_scene] is enabled for all resources on the absolute [NodePath] to the resource.
```
Or similar, unclear what is meant here

Reply:
Wanted to emphasize what happens when a scene is instantiated. Already modified.